rclcpp_lifecycle/include/rclcpp_lifecycle/lifecycle_node.hpp

```diff
@@ -694,13 +694,21 @@ class LifecycleNode : public node_interfaces::LifecycleNodeInterface,
   std::vector<State>
   get_available_states();
 
+  /// Return a list with the current available transitions.
+  /**
+   * \return list with the current available transitions.
+   */
+  RCLCPP_LIFECYCLE_PUBLIC
+  std::vector<Transition>
+  get_available_transitions();
+
   /// Return a list with the available transitions.
```

Comment:
I suggest updating this comment to clarify it returns all the transitions in the transition graph.